client/src/components/Booking.js

```diff
@@ -0,0 +1,162 @@
+import { Avatar, Button, Card, makeStyles } from "@material-ui/core";
+import React, { Fragment, useEffect, useState } from "react";
+import { useProfileContext } from "../contexts/profile";
+
+const useStyles = makeStyles((theme) => ({
+  booking: {
+    display: "grid",
+    gridTemplateColumns: "1fr 3fr 3fr 3fr",
+    gridAutoRows: "auto",
+    padding: 20,
+    alignItems: "center",
+    textAlign: "left",
+    marginTop: 10,
+    [theme.breakpoints.between("xs", "sm")]: {
+      textAlign: "center",
+      gridTemplateColumns: "1fr",
+    },
+  },
+  photo: {
+    gridColumn: "1 / span 1",
+    marginLeft: 0,
+    width: 75,
+    height: 75,
+    [theme.breakpoints.between("xs", "sm")]: {
+      gridRow: "1 / span 1",
+      gridColumn: "1 / span 1",
+      margin: "5px auto"
+    },
+  },
+  name: {
+    gridColumn: "2 / span 1",
+    marginBottom: 10,
+    [theme.breakpoints.between("xs", "sm")]: {
+      gridRow: "2 / span 1",
+      gridColumn: "1 / span 1",
+    },
+  },
+  date: {
+    gridColumn: "3 / span 1",
+    [theme.breakpoints.between("xs", "sm")]: {
+      gridRow: "3 / span 1",
+      gridColumn: "1 / span 1",
+    },
+  },
+  buttons: {
+    gridColumn: "4 / span 1",
+    display: "flex",
+    justifyContent: "center",
+    [theme.breakpoints.between("xs", "sm")]: {
+      gridRow: "4 / span 1",
+      gridColumn: "1 / span 1",
+    },
+  },
+  button: {
+    marginLeft: 10,
+  },
+}));
+
+const formatDate = dateStr => {
```

Comment:
Let's use moment.js and make this more readable

Reply:
Updated in recent commits